server/service/integration_mdm_test.go

```diff
@@ -4712,6 +4711,11 @@ func (s *integrationMDMTestSuite) TestSSO() {
 	// test runs.
 	mdmDevice.EnrollInfo.MDMURL = strings.Replace(enrollURL, "https://localhost:8080", s.server.URL, 1)
 	mdmDevice.EnrollInfo.SCEPURL = strings.Replace(scepURL, "https://localhost:8080", s.server.URL, 1)
+
+	// associate the mdm idp account with the host uuid manually (we can't mock the Apple webview client
+	// deviceinfo header because it is signed with the Apple Root CA)
+	require.NoError(t, s.ds.AssociateMDMIdPAccount(context.Background(), user1EnrollRef, mdmDevice.UUID))
```

Comment:
looking at the source code of the library, we can use https://github.com/korylprince/dep-webview-oidc/blob/2dd846a54fed04c16dd227b8c6c31665b4d0ebd8/header/header.go#L93-L97 to skip verification in tests.

it would have to be an ad-hoc env var or package-level variable like:

```
if os.Getenv("FLEET_TEST_SKIP_VERIFY")
    WithVerify(false)
```

being annoying with this because I feel it's important to have good tests for this flow (It's hard to manually test)

feel free to do it in a follow up